openmc/deplete/abc.py

```diff
@@ -701,6 +712,33 @@ def solver(self, func):
 
         self._solver = func
 
+    @property
+    def reactivity_control(self):
+        return self._reactivity_control
+
+    @reactivity_control.setter
+    def reactivity_control(self, reactivity_control):
+        if isinstance(reactivity_control, CellReactivityController):
+            if not isinstance(reactivity_control.cell, Cell):
+                raise TypeError(
+                    "Reactivity control attribute must be Cell "
+                    "not {}".format(type(reactivity_control.cell)))
```

Comment:
These feel like checks that would belong on the `CellReactivityController` class. Such that it would be impossible to create a cell reactivity controller where the `cell` attribute is not a `Cell`